### DISCUSSION TITLE
Slow build times

### DISCUSSION THREAD

**Discussion by MichaelMauderer** [enso-org/enso#6144] [Help and troubleshooting] [answered]
I recently realized that build times for the IDE had crept up quite a bit again, and I wanted to check whether people are affected by this, too. 

When I make a change to the `GraphEditor` (a notoriously slow to build part of the code base) it can take up to 2-3 minutes for the rust build to finish. 

One thing that I just realized is, that it seems the default command for developing `./run ide/gui watch` now seems to build in release mode, which, of course, takes a fair bit longer than an unoptimized build. Is this on purpose @wdanilo ?

**Comment by wdanilo** [comment]
Regarding optimized build - I was not aware of this change and does not look like done on purpose. It should be not-optimized by default.

**Reply by farmaazon** [reply]
Actually, @mwu-tow _did_ it on purpose (I asked him long ago), to have things consistent.

**Reply by vitvakatu** [reply]
Unoptimized build (`--wasm-profile=dev`) gives me an extremely slow IDE, so I doubt it should be a default.

**Reply by wdanilo** [reply]
Ok, so we have the answer here :) Could you guys post it please as an answer to this thread and mark it as answered, please?

**Comment by farmaazon** [comment, marked as answer]
@mwu-tow did it on purpose (I asked him long ago), for two reasons:
1. This is more consistent (the default value for the option does not depend on other arguments)
2. Unoptimized build (`--wasm-profile=dev`) gives an extremely slow IDE
